python/src/LeastSquaresEquationsSolver_doc.i.in

```diff
@@ -0,0 +1,81 @@
+%feature("docstring") OT::LeastSquaresEquationsSolver
+"Solve systems of non-linear equations using a least squares method.
+
```

Comment:
its missing the experimental warning we usually put here, see other experimental classes
also could you add it to the changelog ?

Reply:
indeed. I realize that for some `Gaussian Process` classes I forgot to add this warning...

Reply:
I can do it in misc if you want